.travis.yml

```diff
@@ -0,0 +1,38 @@
+env:
+  global:
+    - RUST_BACKTRACE=1
+    - PATH=$PATH:$HOME/.cargo/bin
+os:
+  - linux
+  - osx
+language: rust
+rust:
+  - 1.26.1
```

Comment:
We [recently switched](https://github.com/maidsafe/QA/pull/84) to `1.26.2` and `nightly-2018-06-10`. Could you please update it according to the QA docs?